docs/configuration.asciidoc

```diff
@@ -50,6 +50,30 @@ This is a typical `appsettings.json` file that contains some sample configuratio
 }
 ----
 
+[[config-reporter]]
```

Comment:
When I build these changes I get an `element section: validity error` on this line. That's because the ID `config-reporter` is defined twice. It can only be defined once. You'll need to change this ID for the documentation to build.

Reply:
Good catch, thx! I wonder why this was green in the CI 🤔 

If you don't mind I'd go with `[[config-core]]` to keep the naming aligned with the other ones. Already committed.

Reply:
Huh, that's a good point, I'm not sure why the CI was green.

Yes, that name works perfectly!